arrow-libs/core/arrow-autocloseable/src/jvmMain/kotlin/arrow/AutoCloseableExtensions.kt

```diff
@@ -0,0 +1,7 @@
+package arrow
+
+public fun <A : AutoCloseable> AutoCloseScope.install(autoCloseable: A): A =
```

Comment:
I'm struggling to see when this would ever get called vs the member method. Are there some weird shenanigans with how `AutoCloseable` works in common vs jvm?

Reply:
Oh wait. Good point. When AutoCloseable was first introduced it was not a `typealias` on `jvmMain`, so they were not actually the same type but I'm not sure if that's still the case. They were planning to fix this.

Reply:
Okay, so this is currently still needed. https://youtrack.jetbrains.com/issue/KT-55777/Unresolved-kotlin.AutoCloseable-in-JVM

But should removed in Kotlin 2.0